be/src/vec/exec/vexchange_node.cpp

```diff
@@ -73,6 +73,8 @@ Status VExchangeNode::open(RuntimeState* state) {
         RETURN_IF_ERROR(_stream_recvr->create_merger(_vsort_exec_exprs.lhs_ordering_expr_ctxs(),
                                                      _is_asc_order, _nulls_first,
                                                      state->batch_size(), _limit, _offset));
+    } else {
+        ADD_THREAD_LOCAL_MEM_TRACKER(_stream_recvr->mem_tracker());
```

Comment:
Thx for your fix,
Is the crash in the first line of `thread_mem_tracker_mgr.h::add_tracker` `DCHECK(_mem_trackers.find(mem_tracker->id()) == _mem_trackers.end());`?

I'm working on a complete fix for this, and this DCHECK may be commented out shortly.

Reply:
Yes, and I will close the PR since you are working on it.